ludwig/api.py

```diff
@@ -1649,6 +1649,15 @@ def set_logging_level(logging_level: int) -> None:
         else:
             set_disable_progressbar(False)
 
+    @property
+    def config(self) -> Dict[str, Any]:
+        return self.config_obj.to_dict()
```

Comment:
This should have a doc string to make it clear that this returns the config with defaults, like 

"""Returns the fully-rendered config of this model including default values."""